server/controllers/docHandlers.js

```diff
@@ -51,6 +51,34 @@ const retrieveDoc = (req, res) => {
   });
 };
 
+const lockDoc = (req, res) => {
+  Doc.findOne({_id:req.params.documentId}, function(lockerr, lockresult){
```

Comment:
lockErr, lockResult fix for unlockDoc as well

Reply:
ok